lucifer/Shell/__init__.py

```diff
@@ -0,0 +1,63 @@
+"""This package contains the class for the Shell."""
```

Comment:
Codacy found an issue: [First word of the docstring should not be `This` (D404)](https://app.codacy.com/manual/alexspam9090/Lucifer/pullRequest?prid=6170630)